data/translations/ko/state-data.json

```diff
@@ -3,310 +3,410 @@
+    "ip_deadline": "2022년 11월 7일 월요일",
+    "mail_postmarked_deadline": "반드시 2022년 10월 11일 화요일 까지 소인이 찍혀야 합니다.",
+    "default_ip_deadline": "선거당일까지 등록이 가능합니다.",
+    "default_mail_postmarked_deadline": "반드시 선거일 30일 전까지 일자로 소인이 날인되어야 합니다.",
     "confirm_registration_link": "https://sosmt.gov/elections/vote/"
   },
   "ne": {
     "hp_link": "https://sos.nebraska.gov/elections/2020-elections",
     "registration_link": "https://www.nebraska.gov/apps-sos-voter-registration/",
     "more_info_link": "https://sos.nebraska.gov/elections/voter-forms",
-    "ip_deadline": "선거일 기준 2번째 금요일 전",
-    "online_deadline": "선거일 기준 3번째 금요일 전",
-    "mail_postmarked_deadline": "반드시 선거일 기준 3번째 금요일까지 일자로 소인이 날인되어야 합니다.",
+    "ip_deadline": "2022년 10월 28일 금요일",
+    "online_deadline": "2022년 10월 21일 금요일",
+    "mail_postmarked_deadline": "반드시 2022년 10월 21일 금요일 까지 소인이 찍혀야 합니다.",
+    "default_ip_deadline": "선거일 기준 2번째 금요일 전",
+    "default_online_deadline": "선거일 기준 3번째 금요일 전",
+    "default_mail_postmarked_deadline": "반드시 선거일 기준 3번째 금요일까지 일자로 소인이 날인되어야 합니다.",
     "confirm_registration_link": "https://www.votercheck.necvr.ne.gov/voterview"
   },
   "nv": {
     "hp_link": "https://www.nvsos.gov/sos/elections",
     "registration_link": "https://www.nvsos.gov/SOSVoterServices/start.aspx",
     "more_info_link": "https://www.nvsos.gov/sos/elections/voters/registering-to-vote",
-    "ip_deadline": "선거일 기준 4번째 화요일 전",
-    "online_deadline": "조기투표 개시일 전 주 목요일까지 ",
-    "mail_postmarked_deadline": "반드시 선거일 기준 4번재 화요일까지 일자로 소인이 날인되어야 합니다.",
+    "ip_deadline": "2022년 11월 8일 화요일",
+    "online_deadline": "2022년 11월 3일 목요일",
+    "mail_postmarked_deadline": "반드시 2022년 10월 11일 화요일 까지 소인이 찍혀야 합니다.",
+    "default_ip_deadline": "선거일 기준 4번째 화요일 전",
+    "default_online_deadline": "조기투표 개시일 전 주 목요일까지 ",
+    "default_mail_postmarked_deadline": "반드시 선거일 기준 4번재 화요일까지 일자로 소인이 날인되어야 합니다.",
     "confirm_registration_link": "https://www.nvsos.gov/votersearch/"
   },
   "nh": {
     "hp_link": "https://sos.nh.gov/home",
     "more_info_link": "https://sos.nh.gov/elections/voters/register-to-vote/",
-    "ip_deadline": "선거당일까지 등록이 가능합니다.",
+    "ip_deadline": "2022년 11월 8일 화요일",
+    "default_ip_deadline": "선거당일까지 등록이 가능합니다.",
     "confirm_registration_link": "https://app.sos.nh.gov/Public/PartyInfo.aspx"
   },
   "nj": {
     "hp_link": "https://www.state.nj.us/state/elections/index.shtml",
     "registration_link": "https://voter.svrs.nj.gov/register",
     "more_info_link": "https://nj.gov/state/elections/voter-registration.shtml",
-    "online_deadline": "선거일 21일 전까지",
-    "mail_received_deadline": "반드시 선거일 21일 전까지 까지 접수해야 합니다.",
-    "ip_deadline": "선거일 21일 전까지",
+    "ip_deadline": "2022년 10월 18일 화요일",
+    "online_deadline": "2022년 10월 18일 화요일",
+    "mail_postmarked_deadline": "반드시 2022년 10월 18일 화요일 까지 소인이 찍혀야 합니다.",
+    "default_online_deadline": "선거일 21일 전까지",
+    "default_mail_received_deadline": "반드시 선거일 21일 전까지 까지 접수해야 합니다.",
```

Comment:
Updated to `default_mail_postmarked_deadline`

Reply:
fixed